.ci.yaml

```diff
@@ -3180,6 +3180,7 @@ targets:
 
   - name: Mac_x64 build_tests_1_4
     recipe: flutter/flutter_drone
+    presubmit: false
```

Comment:
should we leave a comment why? I could foresee in the future someone cargo-culting this change to the arm64 ones.

Reply:
Added a comment.